packages/app/components/Queue/GroupQuestions.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useMemo } from "react";
+import styled from "styled-components";
+import { Card, Row, Col, Checkbox, Collapse, Button } from "antd";
+import { UserAddOutlined } from "@ant-design/icons";
+
+const { Panel } = Collapse;
+
+const QuestionCard = styled(Card)`
+  background: #fafafa;
+  border: 1px solid #d9d9d9;
+  border-radius: 2px;
+  margin-bottom: 8px;
+`;
+
+const QuestionText = styled.div`
+  font-size: 14px;
+  line-height: 22px;
+  color: #595959;
+`;
+
+const AddButton = styled(Button)`
+  margin-top: 8px;
+`;
+
+const CenterRow = styled(Row)`
+  align-items: center;
+  padding-right: 4px;
+`;
+
+const GroupQuestions = () => {
```

Comment:
can you render these with mock data? an example could be `/mocks/listQuestions.ts`. You don't have to set up an endpoint or anything, but it'd be nice to have the code here to process that mock data so eventually when we set up the endpoint it requries no work

Reply:
yeah for sure! this PR was only intended for the UI though, we're planning on loading mock data for this week's sprint so I'll leave that task for our next PR